package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk",
-  "version": "3.8.3",
+  "version": "3.9.0-alpha.0",
```

```suggestion
  "version": "4.0.0-alpha.0",
```

as mentioned in the commons PR, needs a major version bump. please do it before merging so we don't forget later.

---

Commented on that PR — major bump is not needed as we're changing precomputed client, not the regular one